### PR TITLE
Mental health

### PR DESCRIPTION
**Pull request by peymanmortazavi**

### CODE REVIEW

**Comment by peymanmortazavi** [comment]
@mynameisfiber @wannabeCitizen ready for y'all to review.

File: lib/processors/mentalhealthprocessor.py [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+
+from ..config import CONFIG
+from .lib.utils import process_api_handler
+from .lib.baseprocessor import BaseProcessor
+
+
+class MentalHealthProcessor(BaseProcessor):
+    name = 'mental_health'
+    limit = 200
+    keywords = ["concerned", "important", "love", "confused", "boredom", "greed", "amazing", "enjoyable", "terrible", "angry", "corrupt", "fun", "happiness", "comfort", "shitty", "nothing", "sharing", "privileged", "lucky", "grateful", "difficult", "impossible", "unfair", "exciting", "better", "worse", "upsetting", "easy", "boredom", "confusion", "pride", "confidence", "doubt", "fear", "disappointment", "diligence", "excitement", "sad", "interesting", "sexy", "love", "hate", "anxiety", "anger", "annoyance", "frustration", "empathy", "sympathy", "apathy", "concern", "sadness", "confusion", "condemnation", "anxiety", "enjoyment", "graciousness", "love", "hate", "impatience", "patience"]
```
Comment (mynameisfiber) [inline lib/processors/mentalhealthprocessor.py:13]: Could you break these keywords out into a file (maybe `mentalhealth.txt` either in the processors directory or in the `data/` directory) and load it on init?  You can have the init simply be:

```
def __init__(self):
    with open("data/mentalhealth.txt") as fd:
        self.keywords = [kw.strip() for kw in fd]
    super().__init__()
```

File: lib/processors/mentalhealthprocessor.py [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+
+    def is_good_quote(self, text):
+        text = text.lower()
+        for word in self.keywords:
+            if text.find(word) > 0:
```
Comment (mynameisfiber) [inline lib/processors/mentalhealthprocessor.py:18]: `if word in text:`

**Comment by mynameisfiber** [comment]
@peymanmortazavi i'm not exactly sure what the `process_*` functions do... it seems they return `None` no matter what and don't seem to have any side effects.

**Comment by peymanmortazavi** [comment]
@mynameisfiber I broke them down to many functions so I don't end up with one giant function. it adds to the given "quotes" which is an array. it's an odd way of doing this. I could return it instead but it doesn't matter all that much for this processor. your call though!

**Comment by wannabeCitizen** [comment]
My two cents: if it works, leave it for now. We can clean up later. Right now we have almost too much to do so it's going to be hard to be strict on code.

**Comment by wannabeCitizen** [comment]
@peymanmortazavi this needs cleaned up and made to be up to spec with new changes in the repo. Are you able to clean it up yourself or do you need me to find the time?

**Pull request merged**